include/global.h

```diff
@@ -14,6 +14,8 @@
 
 #define GF_ASSERT(expr) do {if (!(expr)) ErrorHandling();} while (0)
 
+#define UNUSED __attribute__((unused))
+
```

Comment:
this isn't valid, mwcc does not use \_\_attribute\_\_. if you want to use unused, do #pragma unused(var)